tests/code/test_programEligibility.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+from app.models.eventParticipant import EventParticipant
+from app.controllers.events.volunteerRegisterEvent import volunteerRegister
+from peewee import DoesNotExist
+from app.models.user import User
+from app.models.event import Event
+from app.controllers.events.programEligibility import isEligibleForProgram
+from app.models.program import Program
+@pytest.mark.integration
+def test_noUserVolunteerRegister():
```

Comment:
this name doesn't match the function being tested

Reply:
Renamed the test according to its function name